timeless/views.py

```diff
@@ -224,6 +225,51 @@ def get_default_context(self):
         return context
 
 
+class CreateView(GenericView):
+    """ Class which creates objects based on received POST data and provided
+    form class """
+    form_class = None
+    success_view_name = None
+
+    def get_form(self, *args, **kwargs):
+        """ Create form instance """
+        if self.form_class is None:
+            raise NotImplementedError(f"{self.__class__.__name__} must define "
+                                      f"'form_class' attribute")
+        return self.form_class(*args, **kwargs)
+
+    def get_success_url_redirect(self):
+        """ Reverse URL based on view name """
+        if self.success_view_name is None:
+            raise NotImplementedError(f"{self.__class__.__name__} must define "
+                                      f"'success_view_name' attribute")
+        return url_for(self.success_view_name)
+
+    def get_context(self, *args, **kwargs):
+        """ Pass 'from' instance to context if it's not provided
+        (basicaly for 'get' method). """
+        if "form" not in kwargs:
+            kwargs["form"] = self.get_form()
+        return super().get_context(*args, **kwargs)
+
+    def post(self):
+        form = self.get_form(request.form)
+
+        if not form.validate():
+            return self.render_to_response(self.get_context(form=form))
+
+        form.save()
+        return redirect(self.get_success_url_redirect())
+
+
+class UpdateView(GenericView):
+    """ Base view for updating objects"""
```

Comment:
@vryazanov leave puzzles for continue pls

Reply:
@vladarefiev I aleady have jobs in my agenda related to `DeleteView` and `UpdateView`, so I will implement them